selene_sdk/utils/performance_metrics.py

```diff
@@ -247,6 +247,37 @@ def get_feature_specific_scores(data, get_feature_from_index_fn):
     return feature_score_dict
 
 
+def auc_u_test(labels, predictions):
+
+    """
+    Outputs the area under the the ROC curve associated with a certain 
+    set of labels and the predictions given by the training model.
+    Computed from the U statistic.
+
+    Parameters
+    ----------
+    labels: numpy.ndarray
+        Known labels of values predicted by model.
+        Must be one dimensional.
```

Comment:
For this and the other parameter documentation, keep everything on 1 line (only creating a new line if the line exceeds 80 characters)